### ISSUE TITLE
Enable Card Body to center align its contents

### ISSUE THREAD

**Issue by alex-bezek** [cerner/terra-core#1294]
# Issue Description
I have found that in most of our use cases, the contents of a card end up having to utilize text-align center. While this works fine, it ends up with some random css scattered around to do this. 1 option we have to just make a reusable component for centering in a div and pass that to the card body. However, the pattern I have noticed is that cards almost always have their body centered.

My proposal is either Card or Card.Body enable a prop `isContentCentered` which adds this css. 

## Issue Type

- [x] Enhancement

## Expected Behavior
I would like to be able to center a card like this
```
<Card>
  <Card.Body isContentCentered>
     <MyContent />
  </Card.Body>
</Card>
```

## Current Behavior
Currently, a consumer can pass in their content wrapped in a component that applies centering

**Comment by bjankord** [comment]
+1 on `isContentCentered` prop. We could use flexbox to center align or text-align: center. I think like flexbox may better align items in the center as `text-align: center` will only apply to `inline` and inline-ish displays like `inline-block`.

**Comment by alex-bezek** [comment]
good point on the flex box. Who else would need to approve this idea before someone can make the PR? It should be a simple PR and I have people ready to make this centering component in the next few days. I would prefer they just contribute this to terra.

Thanks for the quick response!

**Comment by dm123455** [comment]
I agree with Alex, there are lots of PR where people are adding custom css to align text to the center.
+1 on doing this. @bjankord  @mjhenkes are you fine with it?

**Comment by mjhenkes** [comment]
+1 to `isContentCentered` prop

**Comment by mananjain149** [comment]
I think we need maximum width also in Card Body for some cards like noDataCard.

**Comment by alex-bezek** [comment]
@mananjain149 I'm not convinced the cards size should be a prop on the card itself. It could just be constrained by a container placed around it. You could take that 1 step further and handle center alignment of the card on the page and its now a layout component that can be used for things other than just card as well

![image](https://user-images.githubusercontent.com/8029578/36743052-e1ac299e-1bae-11e8-8f96-dbf7f9b73d4d.png)
![image](https://user-images.githubusercontent.com/8029578/36743062-e9c81dfe-1bae-11e8-9cd6-3b62fdd7f68b.png)

In these examples its not just a card for example

**Comment by emilyrohrbough** [comment]
Resolved by #1300

**Comment by PaulSolDev** [comment]
JIRA created
